samples/spring-boot-auto-config/pom.xml

```diff
@@ -18,6 +18,11 @@
       <groupId>io.javaoperatorsdk</groupId>
       <artifactId>operator-framework-spring-boot-starter</artifactId>
     </dependency>
+    <!-- without this, the app exits after 1 minute  -->
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
Why is it stopping without it? Health check support?

Reply:
see explanation here: https://github.com/operator-framework/josdk-spring-boot-starter/issues/202#issuecomment-2627424838